rotation.py

```diff
@@ -22,6 +22,39 @@ def __str__(self) -> str:
     def __repr__(self) -> str:
         return str(self)
 
+    @staticmethod
+    def is_commute(a: 'PauliOperator', b: 'PauliOperator') -> bool:
```

Comment:
I was thinking we might refactor `is_commute` to be called something like `get_commute_mode`.
Maybe the return type could be an enum like
```
class CommuteMode(Enum):
    Commute = 1
    Anticommute = -1
```

Reply:
I get the point though ... maybe a refactoring for some other time I'd say